code_search/src/code_search/dataflow/transforms/github_dataset.py

```diff
@@ -59,13 +54,11 @@ def expand(self, input_or_inputs):
         lambda row: len(row['docstring_tokens'].split(' ')) > self.min_docstring_tokens)
     )
 
-    if self.target_dataset:
-      logging.info("Writing results to BigQuery %s:%s.%s",
-                   self.project, self.target_dataset, self.pairs_table)
+    if self.pairs_table:
+      logging.info("Writing results to BigQuery %s", self.pairs_table)
       (flat_rows  # pylint: disable=expression-not-assigned
-        | "Save Tokens" >> gh_bq.WriteTokenizedData(self.project, self.target_dataset,
-                                                    self.pairs_table)
+        | "Save Tokens" >> gh_bq.WriteTokenizedData(self.pairs_table)
```

Comment:
Same comment as above about defining a schema and using WriteToBigQuery directly.

Reply:
done